hamilflow/models/kepler_problem/model.py

```diff
@@ -198,8 +199,11 @@ def ecc(self) -> float:
     def period_in_tau(self) -> float:
         """Period in the scaled time tau."""
         if self.ecc >= 1:
+            msg = (
+                f"Only systems with 0 <= eccentricity < 1 have a period, got {self.ecc}"
+            )
             raise TypeError(
-                f"Only systems with 0 <= eccentricity < 1 have a period, got {self.ecc}",
+                msg,
```

Comment:
Is there a particular reason for a new var?

Maybe something like the following is it is about length?

```python
raise TypeError(
    "Only systems with 0 <= eccentricity < 1 "
    f"have a period, got {self.ecc}"
)
```

Reply:
@emptymalei https://docs.astral.sh/ruff/rules/f-string-in-exception/